build-system/compile/compile.js

```diff
@@ -256,8 +256,8 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
       assume_function_wrapper: true,
       // Transpile from ES6 to ES5 if not running with `--esm`
       // otherwise transpilation is done by Babel
-      language_in: 'ECMASCRIPT6',
-      language_out: argv.esm ? 'NO_TRANSPILE' : 'ECMASCRIPT5',
+      language_in: argv.esm ? 'ECMASCRIPT_2017' : 'ECMASCRIPT6',
+      language_out: argv.esm ? 'ECMASCRIPT_2017' : 'ECMASCRIPT5',
```

Comment:
Is this change necessary? 

Why switch to `ECMASCRIPT_2017` which could perform some transpilation when `NO_TRANSPILE` guarantees no additional transpilation from Closure Compiler?

Reply:
no i dont believe this is necessary. was just testing the outputs out. will revert back to NO_TRANSPILE

Reply:
`language_out` should be now `NO_TRANSPILE` when `esm` flag is used